src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1081,9 +1078,9 @@ FixedwingAttitudeControl::task_main()
 
 			} else {
 				/* manual/direct control */
-				_actuators.control[0] = _manual.y;
-				_actuators.control[1] = -_manual.x;
-				_actuators.control[2] = _manual.r;
+				_actuators.control[0] = _manual.y + _parameters.trim_roll;
+				_actuators.control[1] = -_manual.x + _parameters.trim_pitch;
```

Comment:
I'm confused. I change the sign of the pitch trim param in the code and nothing changes.